### PR TITLE
Fix missing co-expression results for miRNAs and phosphoproteins

### PR DESCRIPTION
**Pull request by onursumer**
Fix cBioPortal/cbioportal#8988

## Checks
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

### CODE REVIEW

File: src/pages/resultsView/coExpression/CoExpressionTab.tsx [hunk 1]
```diff
@@ -382,6 +386,16 @@ export default class CoExpressionTab extends React.Component<
...
+                                } else if (
+                                    entrezGeneIdToGene[
+                                        datum.geneticEntityId
+                                    ] !== undefined
+                                ) {
+                                    geneticEntityName =
+                                        entrezGeneIdToGene[
+                                            datum.geneticEntityId
+                                        ].hugoGeneSymbol;
+                                    cytoband = '';
```
Comment (dippindots) [inline src/pages/resultsView/coExpression/CoExpressionTab.tsx:398]: Is it ok to assign empty string to `cytoband`?
```
cytoband = '';
```

**Review by dippindots — COMMENTED** [review]
@onursumer This pr looks good to me, I just have one question regarding to the `cytoband`.

And I'd like to test this fix manually, could you tell me how can I test and which part should I look at? Thanks!

File: src/pages/resultsView/coExpression/CoExpressionTab.tsx (hunk 1 above)
Comment (onursumer) [reply src/pages/resultsView/coExpression/CoExpressionTab.tsx:398]: Good question. It looks empty in the tutorial video https://www.youtube.com/watch?v=YUxVv6pkxD4&t=2240s, but that may not always be the case.

File: src/pages/resultsView/coExpression/CoExpressionTab.tsx [hunk 2]
```diff
@@ -355,6 +356,9 @@ export default class CoExpressionTab extends React.Component<
                         this.props.store.entrezGeneIdToReferenceGene.result ||
                         {};
 
+                    const entrezGeneIdToGene =
```
Comment (alisman) [inline src/pages/resultsView/coExpression/CoExpressionTab.tsx:359]: entrezGeneIdToGeneMap?

Comment (onursumer) [reply src/pages/resultsView/coExpression/CoExpressionTab.tsx:359]: Yeah, makes more sense. We already have another variable called `entrezGeneIdToReferenceGene`. We should probably rename that one as well.

**Review by inodb — COMMENTED** [review]
Seems good to me but would be nice to confirm performance is similar

**Comment by onursumer** [comment]
Failing e2e tests don't seem related, merging.

**Pull request merged**